Userland/Applications/PartitionEditor/main.cpp

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2022, Samuel Bowman <sam@sambowman.tech>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Applications/PartitionEditor/PartitionEditorWindowGML.h>
+#include <Applications/PartitionEditor/PartitionModel.h>
+#include <LibCore/DirIterator.h>
+#include <LibCore/System.h>
+#include <LibGUI/Application.h>
+#include <LibGUI/ComboBox.h>
+#include <LibGUI/ItemListModel.h>
+#include <LibGUI/Menu.h>
+#include <LibGUI/MessageBox.h>
+#include <LibGUI/TableView.h>
+
+using namespace PartitionEditor;
```

Comment:
Please don't

Reply:
Fixed. For reference, I was using TextEditor as a template. Is there a reason some apps (FileManager, Help, ImageViewer, Music, TextEditor) are using their namespaces while the rest don't?

Reply:
Not sure, probably just historical - someone added it, it got missed during code review, and then served as a template for a bunch of other apps.